package.json

```diff
@@ -3,12 +3,13 @@
   "version": "1.0.0",
   "description": "OpenConext-profile provides an overview of your OpenConext profile",
   "dependencies": {
-    "jquery": "^3.6.0",
-    "material-design-lite": "1.0.5"
+    "jquery": "^3.6.0"
   },
   "devDependencies": {
     "@symfony/webpack-encore": "^1.5.0",
-    "file-loader": "^6.2.0"
+    "file-loader": "^6.2.0",
+    "sass": "^1.35.1",
+    "sass-loader": "^12.1.0"
```

Comment:
Can you already get rid of the resolution below? package.json#20

Reply:
ah, i never thought to check, but it appears that yes i can ^__^